.tours/sample-tour.tour

```diff
@@ -0,0 +1,202 @@
+      "line": 96,
+      "selection": {
+        "start": {
+          "line": 79,
+          "character": 2
+        },
+        "end": {
+          "line": 96,
+          "character": 3
+        }
+      }
+    },
+    {
+      "file": "srv/src/main/java/my/bookshop/handlers/CatalogServiceHandler.java",
+      "description": "### Event Handler\n\nThe `onAddReview` method is bound to the `On` phase of the `AddReview` event, as defined by the bound action in the [`CatalogService`](srv/cat-service.cds). Aside from the `Insert` [query execution](https://cap.cloud.sap/docs/java/query-execution#queries) demonstrated here, you can also see how to [complete the event processing](https://cap.cloud.sap/docs/java/provisioning-api#eventcompletion) necessary in the `On` phase. In this example we setting the `Reviews` object from the query result in the event context.\n\nConcepts used:\n\n- [Event Phase: `On`](https://cap.cloud.sap/docs/java/provisioning-api#on)\n- [Event Contexts](https://cap.cloud.sap/docs/java/provisioning-api#eventcontext)\n- [Building CQN Queries](https://cap.cloud.sap/docs/java/query-api)\n- [Static CDS Model](https://cap.cloud.sap/docs/java/advanced#staticmodel)",
+      "line": 123,
+      "selection": {
+        "start": {
+          "line": 98,
+          "character": 2
+        },
+        "end": {
+          "line": 123,
+          "character": 3
+        }
+      }
+    },
+    {
+      "file": "srv/cat-service.cds",
+      "description": "### Bound Action\n\nThe [bound action](https://cap.cloud.sap/docs/cds/cdl#actions) `addReview` here is bound to the `Books` entity within the [`CatalogService`](srv/cat-service.cds). Modeling an action provides you with a more specific [event context](https://cap.cloud.sap/docs/java/provisioning-api#eventcontext) object, such as the `AddReviewContext`, which is used, as shown in the previous step, to implement the `AddReview` event processing.",
```

Comment:
We could link here also to our documentation about how to implement actions and functions, which also describes the action-specific event contexts: https://cap.cloud.sap/docs/java/application-services#actions